flake8diff/exceptions.py

```diff
@@ -1,7 +1,12 @@
 from __future__ import unicode_literals, print_function
 
 
-class Flake8NotInstalledError(Exception):
+class BaseError(Exception):
```

Comment:
might want to use [`@six.python_2_unicode_compatible`](https://pythonhosted.org/six/#six.python_2_unicode_compatible) decorator which will then implement `__unicode__` based on `__str__` in Python 2